flocker/node/agents/blockdevice.py

```diff
@@ -340,3 +504,113 @@ def get_device_path(self, blockdevice_id):
         )
         # May be None if the file hasn't been used for a loop device.
         return _device_for_path(volume_path)
+
+
+def _manifestation_from_volume(volume):
```

Comment:
This on the other hand seems like it might be useful on the `BlockDeviceVolume` class?